tests/cstor-pool/cstor_integration_test.go

```diff
@@ -300,7 +309,7 @@ var _ = Describe("RAIDZ2 SPARSE SPC", func() {
 })
 
 var _ = AfterSuite(func() {
-	spcClient, err := spc.KubeClient().WithFlag(KubeConfigPath)
+	spcClient, err := spc.KubeClient().WithFlag(kubeConfigPath)
```

Comment:
WithFlag seems out of band here?
Does it accept any kind of flag?
cc @shubham14bajpai @mittachaitu

Reply:
The function name should be WithKubeConfig().

Reply:
Renamed the function to `WithKubeConfigPath`